jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -39,6 +39,9 @@ def run(params) {
                     deployed = true
                 }
             }
+            stage('Sanity Check') {
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:sanity_check'"
```

Comment:
`rake cucumber:sanity_check` that might not be the right YAML file.
I would say we need a special one for BV, we need to create a new one like:
`rake cucumber:build_validation_sanity_check` that will point to `spacewalk/testsuite/run_sets/parallel:build_validation_sanity_check.yml` new file.